src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuck.java

```diff
@@ -322,11 +322,14 @@ private void postProcess(Path tmpDir, GoGoDuckModule module) throws GoGoDuckExce
         if (directoryListing != null) {
             for (File file : directoryListing) {
                 try {
+                    logger.info(String.format("Checking file %s for packed variable", file.getAbsolutePath()));
```

Comment:
This is not really whats going on here - perhaps just delete - the two messages below should be sufficient